exercises/hangman/Hangman.cs

```diff
@@ -1,34 +1,45 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Reactive;
+using System.Reactive.Subjects;
 
+/// <summary>
```

Comment:
This comment is very useful to me as a reviewer, but less useful to student I think. As `hangman` is one of the last exercises, a student should at this point be quite familiar with (im)mutability. Therefore, I propose to remove this comment.